src/ddscxx/tests/DataReaderSelector.cpp

```diff
@@ -56,8 +57,11 @@ class DataReaderSelector : public ::testing::Test
         dds::topic::qos::TopicQos topic_qos = this->participant.default_topic_qos();
         topic_qos << dds::core::policy::Reliability::Reliable() <<
                      dds::core::policy::History::KeepAll();
+
+        char topicname[128];
+        create_unique_topic_name("selector_test_topic",topicname,sizeof(topicname));
```

Comment:
This is not a "real" error, but it diverts from the default of 32 that I used before (small nitpick) and it's inconsistent with the 64 you've chosen somewhere else in the PR. It's ok for now, but we should do a follow-up PR that simply change all occurrences to use the same.